spec.bs

```diff
@@ -25,6 +28,8 @@ spec: clear-site-data; urlPrefix: https://www.w3.org/TR/clear-site-data/#
 spec: storage-access; urlPrex: https://privacycg.github.io/storage-access/#
     type: dfn
         text: determine the storage access policy; url: determine-the-storage-access-policy
+urlPrefix: https://w3c.github.io/permissions/; spec: permissions
+    text: permissions task source; url: #permissions-task-source; type: dfn
```

Comment:
And a PR to export this.

Reply:
That's https://github.com/w3c/permissions/pull/407 but they're not super happy with it being exported. There are already bugs for rSA and rSAFor tracking that and I don't think another issue here is helpful, but I should add a comment referencing that issue :)